packages/Dendrogram/src/viewers/tree-renderers/canvas-tree-renderer.ts

```diff
@@ -354,18 +357,7 @@ export class CanvasTreeRenderer<TNode extends MarkupNodeType>
 
           this.selections = selections;
         }
-      } else this.current = this.mouseOver;
-
-      if (!this.current) {
-        this.clickCounter += 1;
-        if (this.clickCounter === 2) {
-          this.clickCounter = 0;
-          this.onResetZoom();
-        }
-        setTimeout(() => {
-          this.clickCounter = 0;
-        }, 300);
-      }
+      } else { this.current = this.mouseOver; }
```

Comment:
you don't need braces here

Reply:
We have the following eslint rule: which makes sure that braces are consistent in blocks. i.e. the above if statement of this 'else' is big so it uses braces and so should this else. I don't think its good idea to remove this rule, we have it in many places.
![image](https://user-images.githubusercontent.com/130361925/235456472-668212eb-e798-4ac6-b4e9-12110a1bd7b2.png)

Reply:
hm. guess we need to discuss this true on code review
because in different places we have different rules, such as writing like `{func}` or like `{ func }`